README.md

```diff
@@ -919,6 +919,42 @@ parsers, it may be more convenient to use the `xml_mode` context manager:
     # ... We have now returned to using the default XML parser.
 ```
 
+Application Cleanup
+-------------------
+
+A user-defined function may be registered to be invoked before, during, or after the time
+when the Python iRODS Client is engaged in object teardown (also called "cleanup") operations.
+This is analogous to the atexit facility in C or in Python itself, except that we have the extra
```

Comment:
Consider linking to `atexit`...

https://docs.python.org/3/library/atexit.html#module-atexit
https://en.cppreference.com/w/c/program/atexit

Probably just the Python one is helpful / relevant here.

Reply:
I'll include both for now. Take a look ... if it seems less relevant , we can probably just delete the C reference.  I was thinking of doing it anyway.

Reply:
Yeah, let's remove the C reference. Providing a link to the python atexit docs gives devs unfamiliar with it a place to read about it.

Reply:
Looks like this is done. Nice

Reply:
Good good.